lib/tasks/dev.rake

```diff
@@ -20,10 +21,20 @@ namespace :openfoodnetwork do
       # -- Shipping / payment information
       unless Spree::Zone.find_by_name 'Australia'
         puts "[#{task_name}] Seeding shipping / payment information"
+
         zone = FactoryGirl.create(:zone, name: 'Australia', zone_members: [])
-        country = Spree::Country.find_by_name('Australia')
         Spree::ZoneMember.create(zone: zone, zoneable: country)
-        FactoryGirl.create(:shipping_method, zone: zone)
+        address = FactoryGirl.create(
+          :address,
+          address1: "15/1 Ballantyne Street",
+          zipcode: "3153",
+          city: "Thornbury",
+          country: country,
+          state: state
+        )
+        enterprise = FactoryGirl.create(:enterprise, address: address)
+
+        FactoryGirl.create(:shipping_method, zone: zone, distributors: [enterprise])
```

Comment:
the enterprise didn't have a valid address specified. That was the reason of the failure.